cli/cli.go

```diff
@@ -10,6 +10,14 @@ import (
 	"github.com/argoproj/pkg/errors"
 )
 
+// SetLogTimestampFormat sets the format of log timestamps
+func SetLogTimestampFormat(format string) {
+	log.SetFormatter(&log.TextFormatter{
+		TimestampFormat: format,
+		FullTimestamp:   true,
```

Comment:
could this be a parameter?

Reply:
@alexec d you mean the whole `Formatter` object? Something like this instead:
```
// SetFormatter sets the logrus log formatter
func SetFormatter(formatter log.Formatter) {
	log.SetFormatter(formatter)
}
```

Reply:
I think he may mean that we might want to have a flag to set `FullTimestamp` on or off

Reply:
got it, thanks!